src/kernel/include/inv.h

```diff
@@ -268,7 +268,7 @@ arcv_deactivate(struct cap_captbl *t, capid_t capin, livenessid_t lid)
  */
 
 static inline void
-sinv_call(struct thread *thd, struct cap_sinv *sinvc, struct pt_regs *regs, struct cos_cpu_local_info *cos_info)
+sinv_call(struct thread *thd, struct cap_sinv *sinvc, struct pt_regs *regs, struct cos_cpu_local_info *cos_info, unsigned long fault_flag)
```

Comment:
`fault_flag` -> `in_fault`

The word `flag` here doesn't denote any information, and is not the normal meaning of `flag` in C.  this is really meant to be a binary value denoting if we are in a fault or not.  `in_fault` (or something like it) should make that more obvious.

This replacement would be done in multiple places, but I'm only going to comment about it here to keep the comments to a more manageable number.